Meetme.ProfileService/Meetme.ProfileService.IntegrationTests/ProfilesControllerTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Meetme.ProfileService.API.ViewModels.ProfileViewModels;
+using Meetme.ProfileService.DAL.Entities;
+using Meetme.ProfileService.IntegrationTests.Common.Attributes;
+using Meetme.ProfileService.IntegrationTests.Common.Keys;
+using Shouldly;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Meetme.ProfileService.IntegrationTests;
+
+public class ProfilesControllerTests : BaseIntegrationTest
+{
+	public ProfilesControllerTests(MeetmeProfileServiceWebApplicationFactory factory) : base(factory)
+	{
+	}
+
+	[Theory, OmitOnRecursionAutoData]
+	public async Task CreateAsync_CreatesProfile(CreateProfileViewModel viewModel)
+	{
+		var response = await httpClient.PostAsJsonAsync(EndPointKeys.Profiles, viewModel, default);
```

Comment:
test methods names do not reflect condition: why or when it creates profile, when it returns bad request and so on..